### PR TITLE
BZ1846118: adding etcd db verification step

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1846118

4.6+

preview: https://deploy-preview-33997--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/backing-up-etcd.html#backing-up-etcd-data_backup-etcd

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@geliu2016, will you PTAL or help me find the right reviewer?

**Comment by geliu2016** [comment]
LGTM, thanks

**Comment by geliu2016** [comment]
Verified code with 4.8.0-0.nightly-2021-06-29-015901

[root@master-00 ~]# /usr/local/bin/cluster-backup.sh /home/core/assets/backup
found latest kube-apiserver: /etc/kubernetes/static-pod-resources/kube-apiserver-pod-8
found latest kube-controller-manager: /etc/kubernetes/static-pod-resources/kube-controller-manager-pod-7
found latest kube-scheduler: /etc/kubernetes/static-pod-resources/kube-scheduler-pod-7
found latest etcd: /etc/kubernetes/static-pod-resources/etcd-pod-2
08c1710131d12a3000fa4a920d6db1db1bbc9d074b1dd203605e1f7842ec5444
etcdctl version: 3.4.14
API version: 3.4
{"level":"info","ts":1625021728.174323,"caller":"snapshot/v3_snapshot.go:119","msg":"created temporary db file","path":"/home/core/assets/backup/snapshot_2021-06-30_025525.db.part"}
{"level":"info","ts":"2021-06-30T02:55:28.181Z","caller":"clientv3/maintenance.go:200","msg":"opened snapshot stream; downloading"}
{"level":"info","ts":1625021728.1819584,"caller":"snapshot/v3_snapshot.go:127","msg":"fetching snapshot","endpoint":"https://145.40.77.203:2379"}
{"level":"info","ts":"2021-06-30T02:55:28.671Z","caller":"clientv3/maintenance.go:208","msg":"completed snapshot read; closing"}
{"level":"info","ts":1625021729.005329,"caller":"snapshot/v3_snapshot.go:142","msg":"fetched snapshot","endpoint":"https://145.40.77.203:2379","size":"94 MB","took":0.830926135}
{"level":"info","ts":1625021729.0054438,"caller":"snapshot/v3_snapshot.go:152","msg":"saved","path":"/home/core/assets/backup/snapshot_2021-06-30_025525.db"}
Snapshot saved at /home/core/assets/backup/snapshot_2021-06-30_025525.db
{"hash":3435630543,"revision":199576,"totalKey":7496,"totalSize":93847552}

File: modules/backup-etcd.adoc [hunk 1]
```diff
@@ -56,20 +56,27 @@ sh-4.4# /usr/local/bin/cluster-backup.sh /home/core/assets/backup
...
+{"level":"info","ts":1624647640.6047094,"caller":"snapshot/v3_snapshot.go:152","msg":"saved","path":"/home/core/assets/backup/snapshot_2021-06-25_190035.db"}
+Snapshot saved at /home/core/assets/backup/snapshot_2021-06-25_190035.db
+{"hash":3866667823,"revision":31407,"totalKey":12828,"totalSize":114446336}
 snapshot db and kube resources are successfully saved to /home/core/assets/backup
 ----
 +
 In this example, two files are created in the `/home/core/assets/backup/` directory on the master host:
 
-* `snapshot_<datetimestamp>.db`: This file is the etcd snapshot.
+* `snapshot_<datetimestamp>.db`: This file is the etcd snapshot. The `cluster-backup.sh` script verifies confirms its validity.
```
Comment (ahardin-rh) [inline modules/backup-etcd.adoc:79]: Looks like there are two verbs here, verifies and confirms, but I assume you just meant to include one.

**Comment by ahardin-rh** [comment]
Just one comment from me. Looks good!

**Pull request merged**